import sys
try:
    import boto3
    print"Sucessfully imported boto3"
except Exception as e:
    print"The error is: ",e
    sys.exit(1)
print"creating a  bucket ..............."
s3 = boto3.resource('s3')
try:
    s3.create_bucket(Bucket="demo_today_bucket",ACL='private') 
    print"demo_bucket is sucessfully created with private permission"
    sys.exit(0)
except Exception as e:
    print"The error is: ",e
    sys.exit(2)
